muon_macros.xml

```diff
@@ -1,9 +1,12 @@
 <macros>
-    <xml name="dftb+">
+    <xml name="dftb_set">
         <param type="select" name="dftb_set" value="3ob-3-1" display="radio" label="DFTB parameter set" help="The parameter set to use for DFTB+. Currently supported are: 3ob-3-1 and pbc-0-3. See help section at the bottom of the page for details.">
```

Comment:
Not a change made here, but where is this information? I cant see anything in the help section.

Reply:
Think this will be a side effect of refactoring into the macros - `pm_yaml_config` and `pm_dftb_opt` have an explaination of the DFTB datasets in their main help, but we don't have that here or in `pm_nq`, but they all import the same xml which references the help text. Will change in both places.

I don't have DFTB installed on my dev VM, if you think there might be an issue I can try and get it set up so I can properly test? There might be a legitimate issue that isn't being caught with the CI.

Reply:
I think it would be worth checking, failing with no explanation wouldn't be great for a user.